src/sanitize-svg.js

```diff
@@ -11,19 +11,22 @@ DOMPurify.addHook(
     'beforeSanitizeAttributes',
     currentNode => {
 
-        if (currentNode && currentNode.href && currentNode.href.baseVal &&
-            currentNode.href.baseVal.replace(/\s/g, '').slice(0, 5) !== 'data:'){
+        if (currentNode && currentNode.href && currentNode.href.baseVal){
```

Comment:
This is super tiny, but: our general code style is to have a space between the closing parenthesis and the opening curly brace
```suggestion
        if (currentNode && currentNode.href && currentNode.href.baseVal) {
```